src/jdk.httpserver/share/classes/sun/net/httpserver/simpleserver/JWebServer.java

```diff
@@ -65,7 +63,7 @@ public static void main(String... args) {
         setMaxReqTime();
         setMaxConnectionsIfNotSet();
 
-        int ec = SimpleFileServerImpl.start(new PrintWriter(System.out, true, UTF_8), "jwebserver", args);
+        int ec = SimpleFileServerImpl.start(new PrintWriter(System.out, true), "jwebserver", args);
```

Comment:
I think you are looking for stdout.encoding here.

Reply:
Are you suggesting changing this line @AlanBateman? The API doc of `PrintWriter(OutputStream, boolean)` says:
```
     * Creates a new PrintWriter from an existing OutputStream.  This
     * convenience constructor creates the necessary intermediate
     * OutputStreamWriter, which will convert characters into bytes using
     * the default charset, or where {@code out} is a {@code PrintStream},
     * the charset used by the print stream.
```
Isn't this what we want here?